src/pages/library/WorkflowsLibrary.tsx

```diff
@@ -0,0 +1,232 @@
+        flexShrink: 0,
+        backgroundImage: `url(${props.logoFilePath})`,
+        backgroundRepeat: 'no-repeat',
+        backgroundPosition: 'center',
+        backgroundSize: 50,
+        width: 67,
+        height: 67,
+        marginRight: 13,
+      }}
+    />
+  );
+};
+
+interface WorkflowSourceBoxProps {
+  title: string;
+  description: string;
+  url: string;
+  logoFilePath: string;
+}
+
+const WorkflowSourceBox = (props: WorkflowSourceBoxProps) => {
+  return (
+    <Clickable href={props.url} {...Utils.newTabLinkProps}>
+      <div
+        style={{
+          width: 400,
+          height: 220,
+          backgroundColor: colors.accent(0.08),
+          padding: '0.8em',
+          borderRadius: '8px',
+          lineHeight: '18px',
+          boxShadow: `3px 3px ${colors.grey(0.2)}`,
+        }}
+      >
+        <div style={{ marginLeft: '10px', marginTop: '4px' }}>
+          <LogoTile logoFilePath={props.logoFilePath} />
+          <div style={{ fontSize: '1rem', fontWeight: 600, color: colors.accent(1.1), marginTop: '5px' }}>
+            {props.title}
+          </div>
+          <p style={{ height: '55px' }}>{props.description}</p>
+          <div style={{ bottom: 0 }}>
+            {icon('pop-out', {
+              size: 18,
+              style: { color: colors.accent(1) },
+            })}
+          </div>
+        </div>
+      </div>
+    </Clickable>
+  );
+};
+
+const CuratedWorkflowsSection = () => {
+  const dockstoreUrlRoot: string = getConfig().dockstoreUrlRoot;
+
+  return (
+    <div>
+      <div style={{ marginTop: '25px' }}>
+        <p style={{ fontSize: '1rem', fontWeight: 600 }}>Curated collections from our community:</p>
+      </div>
+      <div
+        style={{
+          display: 'flex',
+          flexWrap: 'wrap',
+          flexDirection: 'row',
+          height: '100%',
+          position: 'relative',
+        }}
+      >
+        {_.map((wf: CuratedWorkflowDetails) => {
+          return (
+            <Clickable
+              key={wf.key}
+              href={wf.url}
+              {...Utils.newTabLinkProps}
+              style={{
+                ...Style.elements.card.container,
+                width: 150,
+                height: 110,
+                margin: '0 1rem 2rem 0',
+                color: colors.accent(1),
+              }}
+            >
+              <div>{wf.title}</div>
+              <div style={{ bottom: 0 }}>{icon('pop-out', { size: 14, style: { marginLeft: '0.25rem' } })}</div>
+            </Clickable>
+          );
+        }, curatedWorkflowsList(dockstoreUrlRoot))}
+      </div>
+    </div>
+  );
+};
+
+export const WorkflowsLibrary = () => {
+  const dockstoreUrl = `${getConfig().dockstoreUrlRoot}/search?_type=workflow&descriptorType=WDL&searchMode=files`;
+  const workflowsRepoUrl: string = isFeaturePreviewEnabled(FIRECLOUD_UI_MIGRATION)
+    ? Nav.getLink('workflows')
+    : `${getConfig().firecloudUrlRoot}/?return=${getEnabledBrand().queryName}#methods`;
```

Comment:
Will this change to `#workflows` after Adam's PR?

Reply:
Yes. I chatted with Adam and we decided to merge this PR as is and this should be updated in his PR.